changelog.md

```diff
@@ -1,3 +1,38 @@
+### v2.3.1 [view commit logs](https://github.com/marionettejs/backbone.marionette/compare/v2.3.0...v2.3.1)
+
+#### 2.3.1 in overview:
+
+##### Features
+
+* Regions can set a `parentEl` as a way of specifying the DOM tree (default `body`) that they are scoped with. (useful for instance in `LayoutView`).
+
+```js
+  var region = new Region({parentEl: $(“#sub-tree”)})
+```
+
+##### Bug Fixes
+
+* Layout region lookups are now scoped to the layout and not to the entire DOM.
+
+* Calling `delegateEvents` after the `ui` hash has been modified now works.
+
+* Prevent unsetting event listeners on region swap.
+
+* A view's `isDestroyed` state is now explicitly set to false.
+
+##### Refactors
+
+* Added `Marionette._getValue`. This method is similar to `_.result`. If a function is provided we call it with context otherwise just return the value. If the value is undefined return a default value. This method is private and should not be used directly in your code.
+
+* Refactor pass on `collectionView` index tracking to reduce complexity through the use of a ternary.
+
+* Behavior collision resolution was refactored to take advantage of jquery event namespaces.
+
+* Refactor minor partial => bind optimization in behaviors.
+
+* `_.each` iteration is now consistent across the app, there is no need to do `_.each(_.keys(obj), ...)`
```

Comment:
I like noting these changes, sure they are not interesting to everyone but that is why I have them under refactors. They act as a historical marker for us to look back on.